pkg/endpoint/endpoint.go

```diff
@@ -353,7 +353,7 @@ type Endpoint struct {
 	hasSidecarProxy bool
 
 	// LabelsMap is the Set of all security labels used in the last policy computation
-	LabelsMap *identityPkg.IdentityCache
+	LabelsMap *identityPkg.IdentityCache `json:"-"`
```

Comment:
We only access this field in the endpoint package, you can also make the field private.

Reply:
Done. Also renamed the field to be more evocative of what it is used for.